subprojects/wxWidgets.wrap

```diff
@@ -1,5 +1,5 @@
 [wrap-git]
 directory = wxWidgets
 url = https://github.com/wxWidgets/wxWidgets.git
-revision = master
-clone-recursive = true
\ No newline at end of file
+revision = v3.1.4
```

Comment:
Very unfortunate that 2.x doesn't support CMake. I guess this will probably force the move to 3.